website/versioned_docs/version-v3.6.x/mutation.md

```diff
@@ -70,6 +70,7 @@ The `match` section is common to both Assign and AssignMetadata. It supports the
 - namespaces - list of allowed namespaces, only resources in listed namespaces will be mutated
 - namespaceSelector - filters resources by namespace selector
 - excludedNamespaces - list of excluded namespaces, resources in listed namespaces will not be mutated
+- name - the name of an object.  If defined, it matches against objects with the specified name.  Name also supports a prefix-based glob.  For example, `name: pod-*` matches both `pod-a` and `pod-b`.
```

Comment:
Looks like https://github.com/open-policy-agent/gatekeeper/pull/1542 was added for v3.7.0+. Let's remove it for 3.6 docs.

Reply:
Done

Reply:
Ah I think I misunderstood what was meant here. I've updated again.